docs/api-docs/catalog/price-list-overview.md

```diff
@@ -1,32 +1,33 @@
 # Price List API
 <div class="otp" id="no-index">
-	<h3> On This Page </h3>
+	<h3> On this page </h3>
 	<ul>
         <li><a href="#what-is-a-price-list?">What is a Price List?</a></li>
-        <li><a href="#price-list-definitions">Price List Definitions</a></li>
-        <li><a href="#price-list-assignments">Price List Assignments</a></li>
-        <li><a href="#price-list-notes">Price List Notes</a></li>
-        <li><a href="#price-list-resources">Price List Resources</a></li>
+        <li><a href="#price-list-definitions">Price List definitions</a></li>
+        <li><a href="#price-list-assignments">Price List assignments</a></li>
+
+        <li><a href="#price-list-notes">Price List notes</a></li>
+        <li><a href="#price-list-resources">Price List resources</a></li>
 	</ul>
 </div>
 
 ## What is a Price List?
 
 A Price List allows you to populate different versions of catalog pricing and assign them to different [Customer Groups](/api-reference/customer-subscribers/customers-api). The prices are specified exclusively at the variant level.
 
-If an active Price List does not contain prices for a variant then the Catalog pricing will be used. The association of a Price List to a Customer Group can be done either via the Control Panel or using the [Customer Groups API.](/api-reference/customer-subscribers/customers-api)
+If an active Price List does not contain prices for a variant, then the Catalog pricing will be used. You can associate a Price List to a Customer Group via the Control Panel or the [Customer Groups API.](/api-reference/customer-subscribers/customers-api)

```

Comment:
Hi @traci707 and @ektravel!  Another way to make the association here is through price list assignments. 
 https://developer.bigcommerce.com/api-reference/catalog/pricelists-api/price-lists-assignments/createpricelistassignments